OracleWebLogic/samples/wls-K8s/wls-12213-domain/Dockerfile

```diff
@@ -45,8 +45,11 @@ ENV MW_HOME="$ORACLE_HOME" \
 
 # Copy scripts
 # --------------------------------
-USER oracle
+USER root
```

Comment:
I agree with Avi, why do you need to be root to copy the files? Are you having ownership problems running the scripts?